tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "references": [
     { "path": "./packages/design-system-react-native" },
     { "path": "./packages/metamask-module-template" },
-    { "path": "./packages/storybook-react" },
+    { "path": "./docs/storybook-react" },
```

Comment:
Fixing linting issues in typescript 

<img width="559" alt="Screenshot 2024-10-16 at 12 42 08 PM" src="https://github.com/user-attachments/assets/293a7f18-b641-4d69-a015-b325abdb7842">